Kiss.Bff/Extern/RegistryConfig.cs

```diff
@@ -68,17 +68,27 @@ public override void ApplyHeaders(HttpRequestHeaders headers, System.Security.Cl
         }
     }
 
-    public record ZaaksysteemRegistry : RegistryBase
+    public record ZaaksysteemRegistry 
     {
-        public override void ApplyHeaders(HttpRequestHeaders headers, System.Security.Claims.ClaimsPrincipal user)
+        public void ApplyHeaders(HttpRequestHeaders headers, System.Security.Claims.ClaimsPrincipal user)
         {
             var authHeaderProvider = new AuthenticationHeaderProvider(Token, ClientId, ClientSecret);
             authHeaderProvider.ApplyAuthorizationHeader(headers, user);
             headers.Add(CrsHeaderConstants.AcceptCrs, CrsHeaderConstants.Value);
             headers.Add(CrsHeaderConstants.ContentCrs, CrsHeaderConstants.Value);
         }
+
+        public string? ClientId { get; init; }
+        public string? ClientSecret { get; init; }
+        public string? Token { get; init; }
```

Comment:
Volgens mij kunnen we stricter zijn omdat we niet meer de base class inheritten:
```suggestion
        public required string ClientId { get; init; }
        public required string ClientSecret { get; init; }
```